keys/Cargo.toml

```diff
@@ -30,14 +30,14 @@ curve25519-dalek = { version = "4", features = [
 data-encoding = "2.6"
 ed25519-zebra = "4.0"
 hex = "0.4"
+itertools = "0.10.3"
 p256 = "0.13"
 rand = "0.8"
-rand_core = "0.6"
+rand_core = "0.6.4"
```

Comment:
I don't think this is a worthwhile thing to get into. As I've already mentioned privately, we publish a lock file, so a outdated lock file for a contributor is an unlikely problem. What remains is that we're suppose to update the version numbers in `Cargo.toml` each time there's a CVE in any of our direct dependencies. With no real upside, this sounds like unnecessary work we're committing to.

If we just do this because we really want to tick a check box, then … I guess we can do it.

Reply:
We talked about this in a meeting and @jsdanielh reminded me that this might be useful for Rust projects depending on our libraries directly. I missed that and then the change makes sense.